examples/misc/raptor_master.py

```diff
@@ -125,6 +192,7 @@ def result_cb(self, requests):
   # master.wait(count=nworkers)
 
     master.start()
+    master.submit_tasks()
```

Comment:
for this example, do we need to call this method? since workload requests are submitted within `raptor.py` script (raptor:L84-103)